### DISCUSSION TITLE
Why make fullscreen the default display mode?

### DISCUSSION THREAD

**Discussion by leofleeo** [ppy/osu#13417] [Q&A] [answered]
Borderless is like fullscreen but it works with screenshot software e.g greenshot. I've heard that being in fullscreen decreases cursor latency on windows. Is that the reason?

**Comment by peppy** [comment, marked as answer]
doesn't it tell you this in settings already?

**Comment by Gonzalo-Bruna** [comment]
> Borderless is like fullscreen but it works with screenshot software e.g greenshot. I've heard that being in fullscreen decreases cursor latency on windows. Is that the reason?

I thought it was quite obvious because this happens with every game, but don't you notice input lag when not playing with full screen?
